.gitignore

```diff
@@ -9,6 +9,9 @@ lerna-debug.log*
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
+# Example tests
+examples/query.js
```

Comment:
Why is this in .gitignore?

Reply:
Good catch, `query.js` was something that i was using mostly for local testing. I can add it back and showcase examples of tiledb queries.